addons/website_slides/controllers/main.py

```diff
@@ -38,6 +38,15 @@ class WebsiteSlides(WebsiteProfile):
         'date': 'create_date desc',
     }
 
+    @staticmethod
+    def _query_url_keep_bool(path='', path_args=None, **args):
```

Comment:
Isn't it possible to improve QueryURL class instead ? Like adding a ``query_falsy_kw`` (False by default to keep current behavior) that would be integrated directly in the ``__call__`` code ? That way we don't have a custom code on top of QueryURL.